components/org.wso2.carbon.identity.discovery/pom.xml

```diff
@@ -32,6 +32,11 @@
     <url>http://wso2.org</url>
     <dependencies>
         <dependency>
+            <groupId>commons-codec</groupId>
```

Comment:
Is this a test scoped dependency ? if not why we need this dependency  ?

Reply:
This dependency is needed when writing the unit tests for ProviderConfigBuilder class. Otherwise it will give org.apache.commons.codec.binary.Base64 error